pkg/codegen/internal/test/testdata/dash-named-schema/dotnet/Utilities.cs

```diff
@@ -60,6 +62,57 @@ public static InvokeOptions WithVersion(this InvokeOptions? options)
             };
         }
 
+        public static Input<Dictionary<string,T>> ToDict<T>(this InputMap<T> inputList)
```

Comment:
I'd like them in SDK too but I was worried about the rollout order, that is, what if users get upgraded Azure-Native Package before they get an upgraded SDK package? This is not a worry, e.g. we have dep constraints to solve this? I can move to SDK right away then.

Reply:
They'll have to upgrade the core SDK if they want the new invoke syntax. If they don't use it - I think it will just work? I feel like we did these things in the past but I may be wrong.

Reply:
Let's try it I guess.

Reply:
If we're adding it to the SDK, can we name it `ToDictionary` rather than `ToDict` to align with the naming in the framework, e.g. https://source.dot.net/#q=todict?